app-crypt/tpmpolicy/files/tpm_hostpolicy

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
I'd recommend a `set -e` so it aborts if it can't actually read the tpm and so on.